src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -165,15 +181,19 @@ class PillSelectorDropdown extends Component {
                 selector={
                     <Label text={label}>
                         <PillSelector
+                            onChange={noop} // fix console error
```

Comment:
If this component isn't passed onChange in inputProps the following flow error will be thrown:
```
Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`.
    in input (created by PillSelector)
    in span (created by PillSelector)
    in TetherComponent (created by Tooltip)
    in Tooltip (created by PillSelector)
    in PillSelector (created by PillSelectorDropdown)
```